src/deployment/deploylib/configuration.py

```diff
@@ -225,3 +283,38 @@ def update_nsg(
             "proxy_nsg_config": json.dumps(nsg_config),
         },
     )
+
+
+def update_endpoint_params(
+    config_client: InstanceConfigClient,
+    authority: str,
+    client_id: str,
+    tenant_domain: str,
+) -> None:
+
+    config_client.table_service.insert_or_merge_entity(
```

Comment:
Does this go into Azure and updates config table directly in the storage account ?

Reply:
Yes!

Reply:
I think this is a bad idea. I'd argue that  all of the storage updates should go through service. Never direct....

Reply:
Similar behavior has been in place since updating the NSG work - we upload the NSG rules directly. 

I don't disagree that there might be a better methodology, but I'm not sure what that'd be and it'd require changes for both this work and the NSG deployment code. Could be a follow-up PR to figure out another way to upload.

Reply:
I guess since there is existing behavior ....

Reply:
I'll open a work item